theme-data/common/common.json

```diff
@@ -38,6 +38,7 @@
         "secondary": {
           "normal": "@color-white-alpha-00",
           "hover": "@color-white-alpha-07",
+          "pressed": "@color-white-alpha-20",
```

Comment:
I do not find this theme token in the Figma
https://www.figma.com/file/yYfvFEEr3UkW1yUKkW7zEH/Tokens---Theme-Color?node-id=3078%3A6338